tests/test_browser.py

```diff
@@ -1383,6 +1383,15 @@ def test_fs_lz4fs_package(self):
     self.btest(os.path.join('fs', 'test_lz4fs.cpp'), '2', args=['--pre-js', 'files.js', '-s', 'LZ4=1', '-s', 'FORCE_FILESYSTEM'])
     print('    opts')
     self.btest(os.path.join('fs', 'test_lz4fs.cpp'), '2', args=['--pre-js', 'files.js', '-s', 'LZ4=1', '-s', 'FORCE_FILESYSTEM', '-O2'])
+    print('    modularize')
+    self.compile_btest([path_from_root('tests', 'fs', 'test_lz4fs.cpp'), '--pre-js', 'files.js', '-s', 'LZ4=1', '-s', 'FORCE_FILESYSTEM', '-s', 'MODULARIZE=1'])
+    create_test_file('a.html', '''
+      <script src="a.out.js"></script>
+      <script>
+        Module()
+      </script>
+    ''')
+    self.run_browser('a.html', '.', '/report_result?2')
```

Comment:
Is there a reason for using `compile_btest / run_browser`? I think using `btest` as in the previous cases would be simpler.

Reply:
`-s MODULARIZE` requires loading the output javascript in a different way. I'm not very familiar with the test system, but I didn't manage to get it to build with `btest`; all other tests that invoke `-s MODULARIZE` seem to do this

Reply:
Oh, right, we need a custom HTML here, thanks.